deps/zlib/zlib.gyp

```diff
@@ -64,7 +64,7 @@
                 'USE_FILE32API'
               ],
             }],
-            ['(target_arch in "ia32 x64 x32" and OS!="ios") or arm_fpu=="neon"', {
+            ['(target_arch in "ia32 x64 x32" and OS!="ios") or arm_version==7', {
```

Comment:
* Is this a safe assumption that armv7 will have neon support?
* What about armv8 and newer (which always have neon)? Do they get detected as 7?

Reply:
* Yes, see the commit log.

* armv8 is `--dest-cpu=arm64`, armv7 and armv6 are `--dest-cpu=arm`.